x-pack/plugins/searchprofiler/public/application/components/profile_query_editor/profile_query_editor.tsx

```diff
@@ -135,18 +148,40 @@ export const ProfileQueryEditor = memo(() => {
             <EuiSpacer size="s" />
           </EuiFlexItem>
           <EuiFlexItem grow={5}>
-            <EuiButton
-              data-test-subj="profileButton"
-              fill
-              disabled={!licenseEnabled}
-              onClick={() => handleProfileClick()}
-            >
-              <EuiText>
-                {i18n.translate('xpack.searchProfiler.formProfileButtonLabel', {
-                  defaultMessage: 'Profile',
+            {licenseEnabled && !hasIndices ? (
+              <EuiToolTip
+                position="top"
+                content={i18n.translate('xpack.searchProfiler.formProfileButtonTooltip', {
+                  defaultMessage: 'An index must be created before leveraging Search Profiler',
```

Comment:
```suggestion
                  defaultMessage: 'You must have at least one index to use Search Profiler',
```
Attempt to make the sentence active

Reply:
Thank you for your feedback. Changed :)